roles/idm/tasks/configure_idm.yml

```diff
@@ -0,0 +1,38 @@
+---
+
+- name: 'Configure idm inital setup'
+  command: > 
+    ipa-server-install -U 
+      --hostname={{ idm_master_hostname }}
```

Comment:
I'd recommend using the actual hostname for this one - i.e.: defaulting it to `ansible_host` or `inventory_hostname`, but allowing an override. In other words; the role should "auto detect" the content of `idm_master_hostname` when the "first" IdM is provided (potentially has to be *required* - manually - for an "all replica" run).